hotel/config.py

```diff
@@ -30,5 +30,5 @@
 class ExtraConfig:
     @property
     def ONE_WEEK_OR_TAKEDOWN(self):
-        week_from_now = datetime.combine(date.today() + timedelta(days=7), time(23, 59)).replace(tzinfo=c.EVENT_TIMEZONE)
+        week_from_now = c.EVENT_TIMEZONE.localize(datetime.combine(date.today() + timedelta(days=7), time(23, 59)))
```

Comment:
This fixes the ``LMT`` issue.  I was using ``datetime.replace`` instead of ``timezone.localize`` which was a mistake on my part.  (The different is subtle and stupid, bleh.)